src/onboarding/registration/NameAndPicture.tsx

```diff
@@ -50,7 +54,16 @@ function NameAndPicture({ navigation, route }: Props) {
   const asyncKomenciReadiness = useAsyncKomenciReadiness()
   const showGuidedOnboarding = useSelector(showGuidedOnboardingSelector)
   const createAccountCopyTestType = useSelector(createAccountCopyTestTypeSelector)
-  const skipUsername = route.params?.skipUsername //TODO repalce with statsig variable
+  const skipUsername = shouldSkipUsername() //TODO repalce with statsig variable
+  const showAlternatePlaceholder = shouldShowAlternatePlaceholder() //TODO repalce with statsig variable
```

Comment:
Actually, one more thing. This variable should be an enum not a boolean. Its already defined in statsig https://console.statsig.com/4plizaPmWwPL21ASV4QAO0/layers/name_and_picture_screen and in @anthonydaoud 's PR https://github.com/valora-inc/wallet/pull/2950